nextcord/state.py

```diff
@@ -2294,6 +2295,17 @@ def parse_auto_moderation_action_execution(self, data) -> None:
             "auto_moderation_action_execution", AutoModerationActionExecution(data=data, state=self)
         )
 
+    def parse_guild_audit_log_entry_create(self, data) -> None:
+        guild = self._get_guild(int(data["guild_id"]))
+        user_id = int(data["user_id"])
+        user = self.get_user(user_id)
+
+        if guild is not None and user is not None:
+            entry = AuditLogEntry(
+                auto_moderation_rules={}, users={user_id: user}, data=data, guild=guild
+            )
+            self.dispatch("guild_audit_log_entry_create", entry)
+
```

Comment:
It's likely a good idea to dispatch a debug log if the event isn't dispatched due to one of these objects being none

Reply:
That's a good idea, I was using print statements before.